docs/en/reference/sql/ddl/CREATE_TABLE_STATEMENT.md

```diff
@@ -1,40 +1,34 @@
 # CREATE TABLE
+ The `CREATE TABLE` statement is used to create a table. The table name must be unique in one database. 
 
 ## Syntax
 
 ```sql
 CreateTableStmt ::=
     'CREATE' 'TABLE' IfNotExists TableName ( 
       TableElementList CreateTableSelectOpt | LikeTableWithOrWithoutParen ) OnCommitOpt
-
 IfNotExists ::=
     ('IF' 'NOT' 'EXISTS')?
-
 TableName ::=
     Identifier ('.' Identifier)?
     
 TableElementList ::=
     TableElement ( ',' TableElement )*
-
 TableElement ::=
-    ColumnDef
-|   ColumnIndex
+    ColumnDef | ColumnIndex
 ```
 
- The `CREATE TABLE` statement is used to create a table. The table name must be unique if it's in the same database. If the table with the same name is created repeatedly, an error will occur.
 
-The `table_element` list needs to be defined in the table creation statement. `table_element` is divided into column description `ColumnDef` and `Constraint`. OpenMLDB requires at least one ColumnDef in the `table_element` list.
+The `TableElementList` needs to be defined in the `CREATE TABLE` statement. `TableElementList` is consist of column description `ColumnDef` and `Constraint`. OpenMLDB requires at least one `ColumnDef` in the `TableElementList`.
```

Comment:
`TableElementList` is consist of column description `ColumnDef` and `Constraint`
=>
`TableElementList` consists of `ColumnDef` (column definition) and `Constraint`

Reply:
there is no desc about `Constraint`, which is also inconsistent to the Chinese doc (is it ColumnIndex? which is described in the Chinese doc)

Reply:
I will change it to `ColumnIndex`